### PR TITLE
renamed method names in db.go and added a clp for port

### PR DESCRIPTION
**Pull request by RAVAN0407**
Renamed the methods from InitDatabase - InitDB
                                             OpenDatabase - OpenDB 
                                             Ping - PingDB 
Added another method named CloseDB 
Added a Command line argument for port

### CODE REVIEW

**Review by vaibhav-kaushal — CHANGES_REQUESTED** [review]
Small change needed.

File: main.go [hunk 1]
```diff
@@ -30,18 +31,17 @@ func Init() {
...
 	r.POST("/shutdownServiceAsap", controllers.ShutdownService)
 
-	err := resources.InitDatabase()
+	err := resources.InitDB()
 	if err != nil {
 		log.Fatal("E#1KDZRP - " + err.Error())
 	}
 	go dbLogWriter.StartWritingLogs()
-	log.Fatal(fasthttp.ListenAndServe(":8080", r.Handler))
+	log.Fatal(fasthttp.ListenAndServe(os.Getenv("APP_PORT"), r.Handler))
```
[the remark this answers is not in the document]
Comment (RAVAN0407) [reply main.go:46]: Is it fine if i had a default value in the .env file?
or should i handle it here?

[the remark this answers is not in the document]
Comment (RAVAN0407) [reply main.go]: Done!

File: resources/db.go [hunk 2]
```diff
@@ -8,40 +8,26 @@ import (
 	_ "github.com/lib/pq"
 )
 
-// BarkPostgresDb wraps the sqlx.DB in a custom struct to use it as a receiver for query functions
+// BarkPostgresDb wraps the *pgxpool.Pool in a custom struct to use it as a receiver for query functions
 type BarkPostgresDb struct {
 	//Client *sqlx.DB
```
[the remark this answers is not in the document]
Comment (RAVAN0407) [reply resources/db.go]: Yup removed!

File: main.go [hunk 3]
```diff
@@ -30,18 +31,22 @@ func Init() {
...
-	err := resources.InitDatabase()
+	err := resources.InitDB()
 	if err != nil {
 		log.Fatal("E#1KDZRP - " + err.Error())
 	}
 	go dbLogWriter.StartWritingLogs()
-	log.Fatal(fasthttp.ListenAndServe(":8080", r.Handler))
+	port := os.Getenv("APP_PORT")
+	if len(port) == 0 {
+		port = ":8080"
```
Comment (vaibhav-kaushal) [inline main.go:48]: People don't set port numbers as `:8080`. They set it as `8080`

**Review by vaibhav-kaushal — CHANGES_REQUESTED** [review]

File: main.go (hunk 3 above)
Comment (RAVAN0407) [reply main.go]: Made the change!

**Review by vaibhav-kaushal — APPROVED** [review]

**Pull request merged**